README.md

```diff
@@ -2,20 +2,49 @@
 This section describes tools that help you with coding. We provide scripts that allows to
 clone all repositories and install artifacts to Maven local / Maven Central Snapshot repositories.
 
+## Prerequisites on Windows
+Install and configure [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10) (WSL) and configure:
+- Java 8 on WSL
+- Gradle 5+
+- Maven
+
 ## Clone all repositories
 Check a `development/pull-all.sh` script to clone all Knot.x repositories. Please check `-h` option
 for help. Execute this script from `knotx-aggregator/development` directory, executing it from another 
-directory will fail.
+directory will fail. Please check `-r` option for providing custom directory where Knot.x repositories 
+will be cloned, for example `sh pull-all.sh -r ../../knotx` make sure, that provided directory exists. 
 
 ## Build Stack
 Check a `development/build-stack.sh` script to build all cloned repositories. Please check `-h` option
-for help.
+for help. This command should also be executed from `knotx-aggregator/development` directory, in other 
+case it will fail. Please check `-r` option for building Knot.x repositories in custom location, for example
+`sh buils-stack.sh -r ../../knotx` make sure that in the provided directory repositories are properly
+cloned.
 
 Please note that the `knotx-stack` repository contains
 [Gradle composite build](https://docs.gradle.org/current/userguide/composite_builds.html) definition.
 The repository allows to re-build all Knot.x modules and use them during integration tests, bypassing the
 need to publish artifacts to the maven repository first.
 
+After successful build remember to rebuild gradle modules or simply refresh them in your IDE
```

Comment:
The `build-all.sh` command deploys the [Knot.x BOM file](https://github.com/Knotx/knotx-dependencies) (Bill Of Materials)  to the local Maven repository first. The BOM file specifies external dependencies, such as used Vert.x version etc. Then all modules are rebuilt.
After the execution, please refresh your IDE.

Reply:
fixed